10up/insecure-content-warning
# Bulk scan/fix for insecure content from the admin

### Description of the Change



Closes #58

### How to test the Change

* Create a few posts with the following WP cli command `echo '<figure class="wp-block-image size-large"><img src="http://cdn.pixabay.com/photo/2015/06/19/21/24/avenue-815297_960_720.jpg" alt=""/></figure><figure class="wp-block-image size-large"><img src="http://cdn.pixabay.com/photo/2015/06/19/21/24/avenue-815297_960_720.jpg" alt=""/></figure><figure class="wp-block-image size-large"><img src="http://cdn.pixabay.com/photo/2015/06/19/21/24/avenue-815297_960_720.jpg" alt=""/></figure>' | wp post generate --count=1 --post_type=car`
* The newly created posts will contain insecure content (image). In the Admin, go to `Tools -> Insecure Content Warning` and run (and dry-run) the process to fix the newly created posts. Play with all the options.
* Verify the the already existing WP cli command to fix insecure content in posts still works.

### Changelog Entry

> Added - New admin screen to fix insecure content in posts.







### Checklist:
